src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -27,11 +27,11 @@
  * Utility methods used across classes. Just import class to access all methods.
  */
 public final class Utility {
-  // Variables needed to connect to MySQL database.
+  // Variables needed to connect to the local MySQL database.
   public static final String SQL_LOCAL_URL =
       "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
-  public static final String SQL_USER = "root";
-  public static final String SQL_PASSWORD = "";
+  public static final String SQL_LOCAL_USER = "root";
+  public static final String SQL_LOCAL_PASSWORD = "";
```

Comment:
I think you may need to also update all of the other places where either of these variables are used. Otherwise everything else will break!

Reply:
yep, I missed that, done!

Reply:
Make sure you also let Omar and Andrés know - I think they both have PRs that use SQL_USER and SQL_PASSWORD at some point!